File: conditionals/old_enough.py
# Paige Jolley, Old Enough Python

age = float(input("How old are you? \n"))

if age == 15: # <= if always starts the conditional
    print("You can get a learners permit.\n")
elif age == 16: #everything inbetween should be elif
    print("You can drive.\n")
elif age >= 18:
    print("You can vote.\n")
elif age >= 6:
    print("You can go to school\n")
else: # <= else always ends the conditionals
    print("You are young.\n")

File: conditionals/old_enough.c
// Paige Jolley, Old Enough, C
#include <stdio.h>
int age ;

int main(void){
    printf("How old are you?\n");
scanf("%d", &age);
     
if(age == 15){
    printf("You can get a learners permit.\n");
}else if(age == 16){
    printf("You can drive.\n");
}else if(age >= 18){
    printf("You can vote.\n");
}else if(age >= 6){
    printf("You can go to school\n");
}else{
    printf("You are young.\n");
}
    return 0; 
}